src/main/java/uk/gov/pay/publicauth/dao/AuthTokenDao.java

```diff
@@ -84,15 +84,15 @@ public void storeToken(TokenHash tokenHash, TokenLink randomTokenLink, TokenSour
                 handle.createQuery("UPDATE tokens SET revoked=(now() at time zone 'utc') WHERE account_id=:accountId AND token_hash=:tokenHash AND revoked IS NULL RETURNING to_char(revoked,'DD Mon YYYY')")
                         .bind("accountId", accountId)
                         .bind("tokenHash", tokenHash.getValue())
-                        .map(StringMapper.FIRST)
+                        .map(StringColumnMapper.INSTANCE)
```

Comment:
I think this is an appropriate replacement for the deprecated `StringMapper` (not that it matters because we don’t do anything with the result)